plugins/sysinfo/sysinfo.cpp

```diff
@@ -338,7 +340,10 @@ getWmiInfo (int mode)
 				hr = pclsObj->Get (L"Name", 0, &vtProp, 0, 0);
 				break;
 		}
-		WideCharToMultiByte (CP_ACP, 0, vtProp.bstrVal, -1, buffer, SysStringLen (vtProp.bstrVal)+1, NULL, NULL);
```

Comment:
Using `WideCharToMultiByte` here is wrong anyway. It's passed to hexchat_commandf which expects utf-8. Use g_utf16_to_utf8

Also use a dynamic string instead of the fixed 128B buffer with g_strjoin. This will also save you the effort of deciding whether to append ", " or not.